README.md

```diff
@@ -1,5 +1,15 @@
+[![Stars](https://img.shields.io/github/stars/laminlabs/bionty?logo=GitHub&color=yellow)](https://github.com/laminlabs/bionty)
+[![pypi](https://img.shields.io/pypi/v/bionty?color=blue&label=pypi%20package)](https://pypi.org/project/bionty)
+
 # bionty: Basic biological entities
 
 bionty is documented & tested within lamindb: github.com/laminlabs/lamindb.
 
+- Access public ontologies of biological entities
+- Manage synonyms, abbreviations, and hierarchical relationships
+
+## Documentation
```

Comment:
I realize that this section header isn't really necessary neither in lamindb nor here... given how little follows. I'll remove it in lamindb.